c_glib/test/run-test.sh

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-test_dir="$(cd $(dirname $0); pwd)"
+test_dir="$(cd "$(dirname "$0")" || exit; pwd)"
```

Comment:
Can we use `&&` here?

```suggestion
test_dir="$(cd "$(dirname "$0")" && pwd)"
```